packages/redux/src/Oidc.js

```diff
@@ -37,9 +43,13 @@ export const OidcBase = props => {
     configuration,
     notAuthenticated,
     notAuthorized,
+    UserStore,
   } = props;
 
-  if (!isEnabled) {
+  if (isEnabled) {
+    const userManager = authenticationService(configuration, UserStore);
+    loadUser(store, userManager);
```

Comment:
loadUser store will be executed at each render? may be a use Effect is necessary in order to keep code compliant to the older version?

Reply:
Thank again for your awesome PR ^^

Reply:
The render of the full component should be less frequent. But you're right.